hotdoc/core/tree.py

```diff
@@ -162,8 +164,8 @@ def __init__(self, name, generated, project_name, extension_name,
                     key.replace('_', '-').lower(): value}))
             except SchemaError as err:
                 warn('invalid-page-metadata',
-                     '%s: Invalid metadata: \n%s, discarding metadata' %
-                     (self.name, str(err)))
+                     'Invalid metadata in page "%s": %s (discarding metadata)' %
```

Comment:
why ? the idea is to have warnings look like compiler warnings, eg `foo.c: this is very bad`

Reply:
iirc this was hard to read on multiple lines.

Reply:
well I'd rather keep it as is tbh :)